supervisor/api/__init__.py

```diff
@@ -335,6 +335,8 @@ def _register_snapshots(self) -> None:
                     api_snapshots.restore_partial,
                 ),
                 web.get("/snapshots/{snapshot}/download", api_snapshots.download),
+                # Old, remove at end of 2020
+                web.post("/snapshots/{snapshot}/remove", api_snapshots.remove),
```

Comment:
Should we print a deprecation warning if this endpoint is called?

Reply:
I think we should make an blogpost on developers about the API change in this year, like to use the new SUPERVISOR endpoint or soon the API unix socket on container /run - or at least use bashio. There are a lot of outdated stuff around which people just copy&past